### PR TITLE
Magento_Customer: avoid using deprecated escape* methods from Abstrac…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

File: app/code/Magento/Customer/view/adminhtml/templates/system/config/validatevat.phtml [hunk 1]
```diff
@@ -4,15 +4,18 @@
  * See COPYING.txt for license details.
  */
 
-/** @var \Magento\Customer\Block\Adminhtml\System\Config\Validatevat $block */
+/**
+ * @var \Magento\Customer\Block\Adminhtml\System\Config\Validatevat $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Customer/view/adminhtml/templates/system/config/validatevat.phtml]:
```suggestion
/**
 * @var \Magento\Customer\Block\Adminhtml\System\Config\Validatevat $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

File: app/code/Magento/Customer/view/adminhtml/templates/tab/cart.phtml [hunk 2]
```diff
@@ -11,7 +11,7 @@
 <div class="content-header skip-header">
     <table>
         <tr>
-            <td><h4><?= $block->escapeHtml($block->getCartHeader()) ?></h4></td>
+            <td><h4><?= $escaper->escapeHtml($block->getCartHeader()) ?></h4></td>
```
Comment (ihor-sviziev) [inline app/code/Magento/Customer/view/adminhtml/templates/tab/cart.phtml:17]: It seems like escaper is not defined in this file

File: app/code/Magento/Customer/view/adminhtml/templates/tab/view/personal_info.phtml [hunk 3]
```diff
@@ -15,52 +15,52 @@ $allowedAddressHtmlTags = ['b', 'br', 'em', 'i', 'li', 'ol', 'p', 'strong', 'sub
 ?>
 <div class="fieldset-wrapper customer-information">
     <div class="fieldset-wrapper-title">
-        <span class="title"><?= $block->escapeHtml(__('Personal Information')) ?></span>
+        <span class="title"><?= $escaper->escapeHtml(__('Personal Information')) ?></span>
```
Comment (ihor-sviziev) [inline app/code/Magento/Customer/view/adminhtml/templates/tab/view/personal_info.phtml:21]: It seems like escaper is not defined in this file

File: app/code/Magento/Customer/view/adminhtml/templates/tab/view/sales.phtml [hunk 4]
```diff
@@ -11,27 +11,27 @@ $singleStoreMode = $block->isSingleStoreMode();
 <div class="entry-edit fieldset-wrapper">
 
     <div class="fieldset-wrapper-title">
-        <span class="title"><?= $block->escapeHtml(__('Sales Statistics')) ?></span>
+        <span class="title"><?= $escaper->escapeHtml(__('Sales Statistics')) ?></span>
```
Comment (ihor-sviziev) [inline app/code/Magento/Customer/view/adminhtml/templates/tab/view/sales.phtml:17]: It seems like escaper is not defined in this file

File: app/code/Magento/Customer/view/frontend/templates/account/authentication-popup.phtml [hunk 5]
```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var \Magento\Customer\Block\Account\AuthenticationPopup $block */
+/**
+ * @var \Magento\Customer\Block\Account\AuthenticationPopup $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Customer/view/frontend/templates/account/authentication-popup.phtml]:
```suggestion
/**
 * @var \Magento\Customer\Block\Account\AuthenticationPopup $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

File: app/code/Magento/Customer/view/frontend/templates/form/edit.phtml [hunk 6]
```diff
@@ -6,18 +6,21 @@
 
 use Magento\Customer\Block\Widget\Name;
 
-/** @var \Magento\Customer\Block\Form\Edit $block */
+/**
+ * @var \Magento\Customer\Block\Form\Edit $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Customer/view/frontend/templates/form/edit.phtml]:
```suggestion
/**
 * @var \Magento\Customer\Block\Form\Edit $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

File: app/code/Magento/Customer/view/frontend/templates/widget/dob.phtml [hunk 7]
```diff
@@ -5,7 +5,10 @@
  */
 
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
-/** @var \Magento\Customer\Block\Widget\Dob $block */
+/**
+ * @var \Magento\Customer\Block\Widget\Dob $block
```
Comment (ihor-sviziev) [inline app/code/Magento/Customer/view/frontend/templates/widget/dob.phtml]: Please combine all phpdoc blocks to one

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Customer/view/adminhtml/templates/system/config/validatevat.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Customer/view/adminhtml/templates/system/config/validatevat.phtml]: fixed

File: app/code/Magento/Customer/view/adminhtml/templates/tab/cart.phtml (hunk 2 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Customer/view/adminhtml/templates/tab/cart.phtml:17]: fixed

File: app/code/Magento/Customer/view/adminhtml/templates/tab/view/personal_info.phtml (hunk 3 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Customer/view/adminhtml/templates/tab/view/personal_info.phtml:21]: fixed

File: app/code/Magento/Customer/view/adminhtml/templates/tab/view/sales.phtml (hunk 4 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Customer/view/adminhtml/templates/tab/view/sales.phtml:17]: fixed

File: app/code/Magento/Customer/view/frontend/templates/account/authentication-popup.phtml (hunk 5 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Customer/view/frontend/templates/account/authentication-popup.phtml]: fixed

File: app/code/Magento/Customer/view/frontend/templates/form/edit.phtml (hunk 6 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Customer/view/frontend/templates/form/edit.phtml]: fixed

File: app/code/Magento/Customer/view/frontend/templates/widget/dob.phtml (hunk 7 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Customer/view/frontend/templates/widget/dob.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE, WebAPI Tests

**Comment by ihor-sviziev** [comment]
@magento run WebAPI Tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B

**Review by ihor-sviziev — APPROVED** [review]

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B

**Pull request closed without merging**